src/app/wysiwyg-editor/terra-wysiwyg-editor.component.ts

```diff
@@ -0,0 +1,72 @@
+import {
+    Component,
+    EventEmitter,
+    Input,
+    OnInit,
+    Output
+} from '@angular/core';
+import { TranslationService } from 'angular-l10n';
+import {
+    ControlValueAccessor,
+    NG_VALUE_ACCESSOR
+} from '@angular/forms';
+
+@Component({
+    selector:  'terra-wysiwyg-editor',
+    template:  require('./terra-wysiwyg-editor.component.html'),
+    styles:    [require('./terra-wysiwyg-editor.component.scss')],
+    providers: [{
+        provide:     NG_VALUE_ACCESSOR,
+        useExisting: TerraWysiwygEditorComponent,
+        multi:       true
+    }]
+})
+export class TerraWysiwygEditorComponent implements OnInit, ControlValueAccessor
+{
+    @Input() inputPlaceholder:string;
+
+    @Output() ngModelChange:EventEmitter<string> = new EventEmitter();
+
+    private _placeholder:string;
+    private _value:string;
+    private _onChangeCallback:() => void;
+    private _onTouchedCallback:() => void;
+
+    constructor(private translation:TranslationService)
+    {
+        // initialize placeholder
+        this._placeholder = this.translation.translate('terraEditor.insertText');
```

Comment:
vielleicht bei ngOnInit die If-Anweisung um einen Else-Zweig ergänzen und dort diese Anweisung verschieben, da sie logisch zusammen gehören.
im Test konnte ich den standard-text für den Placeholder nicht lesen. es stand einfach: terraEditor.insertText

Reply:
Mit dem else-Zweig das ist Geschmackssache. Ich definiere gerne generell einen Standardfall und überschreibe ihn in speziellen Fällen (if).
Die Übersetzung funktioniert im Terra-Components-Projekt nicht. Wenn du sie mal in Terra einbindest, dann sollte der Standard-Placeholder angezeigt werden.

Reply:
👍